src/client.js

```diff
@@ -1654,16 +1654,41 @@ class Client {
     async getSchema(schemaId, representation, internal) {
         var entity = await this._entityCache.get("xtk:schema", schemaId);
         if (!entity) {
-          entity = await this.getEntityIfMoreRecent("xtk:schema", schemaId, "xml", internal);
-          if (entity) {
-            const impls = DomUtil.getAttributeAsString(entity, "implements");
-            if (impls === "xtk:persist" && schemaId !== "xtk:session" && schemaId !== "xtk:persist") {
-                // Ensure xtk:persist is present by loading the xtk:session schema
-                await this.getSchema("xtk:session", "xml", true);
+              // special case of temp schema for nms:group
+              if (schemaId.startsWith("temp:group:")) {
```

Comment:
We'll need documentation for this new type of schemas

Reply:
Yes, I mean a kind of schema which was not previously handled by the SDK. So it should still be documented. As we also take the opportunity of this SDK to document obscure thinks in Campaign. In this case, it would be great to have a small paragraph explaining what schemas starting with temp:group: are, how they are constructed, and give examples.